go.mod

```diff
@@ -136,6 +136,13 @@ exclude (
 	github.com/coreos/bbolt v1.3.3
 	github.com/etcd-io/bbolt v1.3.3
 	github.com/etcd-io/bbolt v1.3.6
+	// exclude github.com/golang/protobuf < 1.3.2 https://nvd.nist.gov/vuln/detail/CVE-2021-3121
+	github.com/gogo/protobuf v1.0.0
+	github.com/gogo/protobuf v1.1.1
+	github.com/gogo/protobuf v1.2.0
+	github.com/gogo/protobuf v1.2.1
+	github.com/gogo/protobuf v1.3.0
+	github.com/gogo/protobuf v1.3.1
```

Comment:
There are more [tags](https://github.com/gogo/protobuf/tags): do we need to exclude rest of them? E.g. `v0.5`, `v0.4`, etc